4-forms-validations/form.html

```diff
@@ -16,19 +16,19 @@ <h1>Form</h1>
         <form>
             <fieldset>
```

Comment:
You can put the `<fieldset>` to group the `gender` as you did with the `socioeconomic status`  and the `hobbies`.